vtds_cluster_kvm/private/cluster.py

```diff
@@ -244,7 +245,7 @@ def __cluster_node_count(self):
         ]
         return sum(counts)
 
```

Comment:
REVIEWERS: added the `config` parameter to make it clearer what this is doing (i.e. eliminate an apparent side-effect from the function). I did this to several functions so that the fact that `self.config` is being modified would be explicit instead of implicit.